client/src/app/mainwindow.cpp

```diff
@@ -428,6 +428,8 @@ void MainWindow::createTabs()
     mUi.tabWidget->addTab(page, i18n("&Opportunities"));
 
     connect(page, SIGNAL(modelCreated(ItemsTreeModel*)), this, SLOT(slotOppModelCreated(ItemsTreeModel*)));
+    connect(static_cast<AccountsPage*>(mAccountPage), &AccountsPage::requestNewOpportunity,
+            static_cast<OpportunitiesPage*>(page), &OpportunitiesPage::createOpportunity);
```

Comment:
Use a new local variable of type OpportunitiesPage for this particular page, to avoid the need to cast.
 (and therefore, move "Page *page" further down)

Reply:
done